docs/api/graphql/graphql_custom_ft.md

```diff
@@ -0,0 +1,163 @@
+---
+description: Add custom Field Types to GraphQL.
+---
+
+# Add custom Field Types to GraphQL
+
+If you want to use custom Field Types in GraphQL, you need to map them.
+Their values and field definition structure, need to be defined, for GraphQL to read them.
+For example:
+
+| Name | Possible field value      | Resolver| Field definition|
+|------|------| ------- | ------- |
+| Text Line     | string | default | `TextLineFieldDefinition`    |
+| Relation List | `Item` `ArticleItem` `ImageItem` | customized | `RelationListFieldDefinitio` |
+
+## Map a custom Field Type
+
+There are two ways to map a custom Field Type:
+
+- configuration
+- custom `FieldDefinitionMapper`
+
+If the mapping of your Field Type depends on the field definition, you need to write a custom `FieldDefinitionMapper`.
```

Comment:
I'd actually say with "if the behaviour of...". We could also add an example: "For example, the selection field type has a "multiple" option. If set to false, it will accept and return a single value, but if set to true, it will accept and return an array of values."